tools/ci/docker/engine/docker-entrypoint.sh

```diff
@@ -17,5 +17,9 @@ PROFILING_OPTIONS=""
 if [ "$PROFILING_FILENAME" != "" ] && [ "$PROFILING_TIME" != "" ]; then
   PROFILING_OPTIONS="--profiling-path /opt/enso/profiling/$PROFILING_FILENAME --profiling-time=$PROFILING_TIME"
 fi
+NATIVE_PROP="--jvm"
+if [ "$ENSO_LAUNCHER" != "" ] && [ "$ENSO_LAUNCHER" == "native" ]; then
+  NATIVE_PROP="-Dcom.oracle.graalvm.isaot=true"
```

Comment:
- manually setting this property is unfortunate
- it is probably result of [this request of mine](https://github.com/enso-org/enso/pull/11880#pullrequestreview-2528578132)
- where I wanted to [drop dependency on Truffle API](https://github.com/enso-org/enso/pull/11880#discussion_r1901489361)
- we have already faced that [issue once here](https://github.com/enso-org/enso/pull/11880/files#r1905115563)
- I assume we are doing something wrong (per my guidance)

Reply:
Yes, the primarily reason for using the property is unwillingness to add dependencies.

Reply:
> unwillingness to add dependencies

- the dependencies are already included
- #12445 doesn't add any new dependency
- we seem to be using `com.oracle.graalvm.isaot` improperly
- better to use `ImageInfo` as #12445 demonstrates